### DISCUSSION TITLE
Can anyone help me identify this camera?

### DISCUSSION THREAD

**Discussion by SnowmanDK** [alienatedsec/yi-hack-v5#393] [Q&A] [answered]
I am trying to identify my camera, so I can find out if it's even possible to apply one of the yi-hacks available.
I have this on the box it came in:
![IP Cam Box - anon](https://github.com/user-attachments/assets/07afcbfd-7b30-47b6-a37b-b58dadd42277)
The firmware is here (in the YI IOT app):
![IPCam Firmware](https://github.com/user-attachments/assets/3916b0c5-acc5-453b-8e18-f07c0868b799)
I don't know if this is useful, but I found this document online:
![IP Cam Model-Difference-7131077](https://github.com/user-attachments/assets/d4cf274b-dcdd-43aa-b790-3b7d78ba97b6)

I have not been able to find my camera model anywhere (maybe I'm doing it wrong).
I really hope someone can help, as I really would like to have it support SRTP/ONVIF, so if ANYONE can tell me which yi-hack to get, I would REALLY appreciate it.

Thanks in advance
SnowmanDK

**Comment by SnowmanDK** [comment]
No one?

**Comment by SnowmanDK** [comment]
Is there life?

**Reply by alienatedsec** [reply]
To identify the camera, you need to disassembly it. However, its likely a Anyka model-based.

https://github.com/roleoroleo/yi-hack-Allwinner-v2/discussions/496

**Reply by SnowmanDK** [reply]
I opened it, and took this picture. How do I know which hack to use?
![IMG_20240726_191950361](https://github.com/user-attachments/assets/377b6500-5e53-47c6-8f58-5f418b6c719b)

**Reply by alienatedsec** [reply]
@SnowmanDK Search for Anyka AK3918E hack. Not sure if any exists.

**Comment by SnowmanDK** [comment, marked as answer]
Just to let others know...
I found a working hack for my camera here: [https://github.com/MuhammedKalkan/Anyka-Camera-Firmware](https://github.com/MuhammedKalkan/Anyka-Camera-Firmware)
